src/EditorFeatures/CSharpTest/ConvertToInterpolatedString/ConvertConcatenationToInterpolatedStringTests.cs

```diff
@@ -322,5 +322,26 @@ void M()
     }
 }");
         }
+
+        [WorkItem(16820, "https://github.com/dotnet/roslyn/issues/16820")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertToInterpolatedString)]
+        public async Task TestWithMultipleStringConcatinations()
```

Comment:
We don't trigger if we detect this case.  I've added a test to make sure this remains the case.